docs/examples/userguide/early_binding_for_speed/rectangle_cpdef.pyx

```diff
@@ -15,5 +15,5 @@ cdef class Rectangle:
         return area
 
 def rectArea(x0, y0, x1, y1):
-    rect = Rectangle(x0, y0, x1, y1)
+    cdef rect = Rectangle(x0, y0, x1, y1)

```

Comment:
This should also say `cdef Rectangle rect` in order to be effective.

Reply:
Oh yes, of course! Seems like these 20 lines of Cython have a curse on them :smiley:

Reply:
I fixed it, by the way